### PR TITLE
utility breakpoints

### PR DESCRIPTION
**Pull request by AlecRaeside**

### CODE REVIEW

File: utilities/_u-float.scss [hunk 1]
```diff
@@ -11,11 +11,29 @@
  */
 
 
+/**
+ * Settings.
+ */
+
+$float-breakpoints: $default-utility-breakpoints !default;
```
Comment (chris-pearce) [inline utilities/_u-float.scss:19]: Can we comment: `// Apply at these breakpoints`?

File: utilities/_u-gpu-accelerated.scss [hunk 2]
```diff
@@ -13,7 +13,6 @@
  * http://codepen.io/team/westfieldlabs/full/xFHfk#gpu-accelerated
  */
 
-
```
Comment (chris-pearce) [inline utilities/_u-gpu-accelerated.scss:16]: This blank line is needed. Need 2 blank lines after intro comments.

File: utilities/_u-image-replacement.scss [hunk 3]
```diff
@@ -11,6 +11,12 @@
  */
 
 
+/**
+ * Settings.
+ */
+
```
Comment (chris-pearce) [inline utilities/_u-image-replacement.scss:17]: Can we comment: `// Apply at these breakpoints`?

File: utilities/_u-image-replacement.scss [hunk 4]
```diff
@@ -11,6 +11,12 @@
...
+$image-replacement-breakpoints: $default-utility-breakpoints !default;
+
```
Comment (chris-pearce) [inline utilities/_u-image-replacement.scss:20]: Need 2 blank lines between these comments:

```
/**
 * Settings.
*/
```

File: utilities/_u-link-complex.scss [hunk 5]
```diff
@@ -23,6 +23,13 @@
 
 
 /**
+ * Settings.
+ */
+
```
Comment (chris-pearce) [inline utilities/_u-link-complex.scss:28]: Need the comment (see above)

File: utilities/_u-link-disguised.scss [hunk 6]
```diff
@@ -11,6 +11,12 @@
  */
 
 
+/**
+ * Settings.
+ */
+
```
Comment (chris-pearce) [inline utilities/_u-link-disguised.scss:17]: Comment

Comment (chris-pearce) [inline utilities/_u-link-disguised.scss:17]: 2 blank lines.

File: utilities/_u-list-block.scss [hunk 7]
```diff
@@ -23,6 +23,9 @@
  * Settings.
  */
 
+ // Apply at these breakpoints
+ $list-block-breakpoints: $default-utility-breakpoints !default;
```
Comment (chris-pearce) [inline utilities/_u-list-block.scss]: Indent var value to match the others.

File: utilities/_u-list-block.scss [hunk 8]
```diff
@@ -23,6 +23,9 @@
  * Settings.
  */
 
+ // Apply at these breakpoints
```
Comment (chris-pearce) [inline utilities/_u-list-block.scss]: Remove this space

File: utilities/_u-list-block.scss (hunk 7 above)
Comment (chris-pearce) [inline utilities/_u-list-block.scss]: Remove this space

File: utilities/_u-list-block.scss [hunk 9]
```diff
@@ -47,38 +50,61 @@ $list-block-border-thickness:     1 !default;
...
-.u-list-block--tiny > li {@include to-rem(padding, $list-block-item-padding-tiny);}
+.u-list-block--tiny > li {@include to-rem(padding,
+  $list-block-item-padding-tiny);}
 
+@include generate-at-breakpoints('.u-list-block--tiny',
+  $list-block-breakpoints) {
+  @include to-rem(padding, $list-block-item-padding-tiny);
+}
 
 /**
```
Comment (chris-pearce) [inline utilities/_u-list-block.scss:72]: You want 2 line breaks between the Modifiers

File: utilities/_u-list-inline.scss [hunk 10]
```diff
@@ -74,11 +78,22 @@ $list-inline-divider-thickness:    1 !default;
 .u-list-inline--spacing-base {
   > li + li {@include to-rem(margin-left, $list-inline-item-spacing-base);}
 }
+@include generate-at-breakpoints('.u-list-inline--spacing-base',
```
Comment (chris-pearce) [inline utilities/_u-list-inline.scss:83]: Blank space needed between this

File: utilities/_u-float.scss (hunk 1 above)
Comment (AlecRaeside) [inline utilities/_u-float.scss:19]: isnt that only if there are multiple comments?

File: utilities/_u-list-inline.scss [hunk 11]
```diff
@@ -154,9 +218,15 @@ $list-inline-divider-thickness:    1 !default;
...
-  > li + li {@include to-rem(border-left, $list-inline-divider-thickness solid $list-inline-divider-colour);}
+  > li + li {@include to-rem(border-left, $list-inline-divider-thickness
+    solid $list-inline-divider-colour);}
 }
 
+@include generate-at-breakpoints('.u-list-inline--divider',
+  $list-inline-breakpoints) {
+  > li + li {@include to-rem(border-left, $list-inline-divider-thickness
+    solid $list-inline-divider-colour);}
+}
```
Comment (chris-pearce) [inline utilities/_u-list-inline.scss:232]: 2 blank lines

File: utilities/_u-list-inline.scss [hunk 12]
```diff
@@ -212,24 +299,65 @@ $list-inline-divider-thickness:    1 !default;
...
   // Huge
   %u-list-inline--spacing-huge.u-list-inline--delimited-slash > li + li:before,
-  .u-list-inline--spacing-huge.u-list-inline--delimited-slash > li + li:before {@include to-rem(margin-right, $list-inline-item-spacing-huge);}
+  .u-list-inline--spacing-huge.u-list-inline--delimited-slash > li + li:before
+  {@include to-rem(margin-right, $list-inline-item-spacing-huge);}
+
+  @include generate-at-breakpoints(
+    '.u-list-inline--spacing-huge.u-list-inline--delimited-slash{bp} > li +
+     li:before', $list-inline-breakpoints) {
+    @include to-rem(margin-right, $list-inline-item-spacing-base);
```
Comment (chris-pearce) [inline utilities/_u-list-inline.scss]: `$list-inline-item-spacing-base` is wrong.

File: utilities/_u-list.scss [hunk 13]
```diff
@@ -45,14 +48,17 @@ $list-bullet-ordered:										decimal !default;
 	// Left indent including nested lists
 	&,
 	ul,
-	ol {
-		@include to-rem(margin-left, $list-indent);
+	ol {@include to-rem(margin-left, $list-indent);}
```
Comment (chris-pearce) [inline utilities/_u-list.scss]: why use `$list-indent`?

File: utilities/_u-list-block.scss [hunk 14]
```diff
@@ -23,6 +23,9 @@
  * Settings.
  */
 
+// Apply at these breakpoints
+$list-block-breakpoints:         $default-utility-breakpoints !default;
```
Comment (chris-pearce) [inline utilities/_u-list-block.scss]: var value isn't aligned.

**Comment by chris-pearce** [comment]
LGTM

**Pull request merged**